README.md

```diff
@@ -18,3 +18,16 @@ Additionally we need the `@zkopru/client` package. This needs to be linked local
 ## Development
 
 A development server with hot reload can be started with `npm run dev` it will be accessible at `localhost:8080`.
+
+## Development alongside @zkopru/client
+
+the web wallet download `@zkopru/client` module from npm while installing. for more convenience to develop alongside @zkopru/client, It would be better to use a client package that is locally cloned.
```

Comment:
question about this, it works well by using `yarn link @zkopru/client` as mentioned above Line16.  Why do we need to have another command `npm run install-dev` for `zkopru/client` integration?

Reply:
In my environment, `yarn link` requires permission to create a folder in the global node module.

I think it is okay setup and forgets in this way. If you use `yarn link`, have to unlink at some point, even though not worry about the permission.

Reply:
May need some more explanation in this section about using script or `yarn link` :)